### PR TITLE
test: Test to check year in license file

### LINKED ISSUE #61: Add a unittest to check the license.md file date range
The end year in the license file should be "this year" and this should be checked on every run of the tests. If the end year is no correct, fail the test.
example: 
> Copyright (c) 2012-2016 SendGrid, Inc.
It is 2017, so this should be:
> Copyright (c) 2012-2017 SendGrid, Inc.
and the  test should fail.
--done

### PR DESCRIPTION
**Pull request by pushkyn**
# Fixes #61 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

File: src/test/java/com/sendgrid/LicenseTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Calendar;
+
+public class LicenseTest {
+
+    @Test
+    public void testLicenseShouldHaveCorrectYear() throws IOException {
+        String copyrightText = null;
+        try (BufferedReader br = new BufferedReader(new FileReader("./LICENSE.txt"))) {
```
Comment (mithunsasidharan) [inline src/test/java/com/sendgrid/LicenseTest.java]: invalid file name

Comment (pushkyn) [reply src/test/java/com/sendgrid/LicenseTest.java]: fixed, thanks

**Comment by mbernier** [comment]
Thank you!

**Comment by thinkingserious** [comment]
Hello @pushkyn,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**